sklearnex/tests/test_patching.py

```diff
@@ -14,107 +14,278 @@
 # limitations under the License.
 # ==============================================================================
 
+
+import importlib
 import inspect
+import io
```

Comment:
Please remove non used import.
Unfortunately, linter sometime misses this to check,
```suggestion
```